frontend/src/modules/dataset/dataset-browser.reducer.tsx

```diff
@@ -147,16 +147,7 @@ export type DatasetBrowserState = {
 export const getDatasetContents = createAsyncThunk(
     'dataset/list_files',
     async (props: DatasetServerRequestProps) => {
-        const { datasetContext, projectName, username } = props;
-        let scope: string | undefined = datasetContext.type;
-
-        switch (datasetContext.type) {
-            case DatasetType.PROJECT:
-                scope = projectName;
-                break;
-            case DatasetType.PRIVATE:
-                scope = username;
-        }
```

Comment:
Are you super-extra-100%-definitely sure you can remove this? Like I think [here](https://github.com/awslabs/mlspace/blob/develop/frontend/src/modules/dataset/dataset-selector.tsx#L52-L55) it is called with the results from [datasetFromS3Uri](https://github.com/awslabs/mlspace/blob/develop/frontend/src/shared/util/dataset-utils.ts#L21) in which scope might be empty for the global scope. So I think unless we modify `datasetFromS3Uri` (which might be possible) we can't assume `.scope` is always defined.

Reply:
Hmm I think this was also from me tweaking the browsers/selectors, will revert this